# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/worklet/WorkletGlobalScope.h
# Commit: a8e623b228ae
# Full Hash: a8e623b228aeb04c741bebeeaea4f2524b2ddcc7
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2022-11-29 21:47:49
# Regressor Bug: 1778510
# File Overlap Count: 1
# Description:
#   Bug 1778510: Add RTPCallerType to GlobalObject r=asuth
#   
#   This centralizes the logic in one place.
#   
#   In order to do this, we will need to check the principal
# ==============================================================================

diff -r 32fdf6f014f7 -r a8e623b228ae dom/worklet/WorkletGlobalScope.h
--- a/dom/worklet/WorkletGlobalScope.h	Tue Nov 29 13:34:14 2022 +0000
+++ b/dom/worklet/WorkletGlobalScope.h	Tue Nov 29 13:34:15 2022 +0000
@@ -73,6 +73,8 @@
  protected:
   ~WorkletGlobalScope();
 
+  bool IsSystemPrincipal() const override;
+
   const RefPtr<WorkletImpl> mImpl;
 
  private: